src_python/org_agg.py

```diff
@@ -0,0 +1,238 @@
+"""
+Copyright 2020 JasmineGraph Team
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import socket
+import pickle
+import select
+import time
+import numpy as np
+import pandas as pd
+import sys
+import logging
+from timeit import default_timer as timer
+import gc
+import math
+import os
+
+# CONSTANTS
+HEADER_LENGTH = 10
+
+logging.basicConfig(
+    level=logging.INFO, 
+    format='%(asctime)s : [%(levelname)s]  %(message)s',
+    handlers=[
+        logging.FileHandler('aggregator.log'),
+        logging.StreamHandler(sys.stdout)
+    ]
+)
+
+class Aggregator:
+
+    def __init__(self, model, rounds , graph_id, num_orgs , ip, port):
+
+        # Parameters
+        self.IP = ip
+        self.PORT = port
+        self.NUM_CLIENTS = num_orgs
+        self.ROUNDS = rounds
+
+        # Global model
+        self.model_weights = model
+
+        self.weights = []
+        self.partition_sizes = []
+        self.training_cycles = 0
+
+        self.stop_flag = False
+
+        # List of sockets for select.select()
+        self.sockets_list = []
+        self.clients = {}
+        self.client_ids = {}
+
+        # Craete Aggregator socket
```

Comment:
Change `Crate` to `Create`?

Reply:
Fixed in 1875888487e22c8e17cbadd8acc558b28d98ebf0

Reply:
Done